.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: composer install --no-interaction --optimize-autoloader
 
       - name: Setup database
-        run: bin/console doctrine:schema:create
+        run: bin/console doctrine:migrations:migrate --no-interaction
```

Comment:
Why would I like this? 

It just takes more time, right? 

Is it because we want to make sure we have not forgotten to add migrations?

Reply:
> Is it because we want to make sure we have not forgotten to add migrations?

Yes

Reply:
Okey. Fair.